common/lib/xmodule/xmodule/poll_module.py

```diff
@@ -215,15 +215,15 @@ def definition_from_xml(cls, xml_object, system):
 
     def definition_to_xml(self, resource_fs):
         """Return an xml element representing to this definition."""
-        poll_str = u'<{tag_name}>{text}</{tag_name}>'.format(
+        poll_str = HTML('<{tag_name}>{text}</{tag_name}>').format(
             tag_name=self._tag_name, text=self.question)
         xml_object = etree.fromstring(poll_str)
         xml_object.set('display_name', self.display_name)
 
         def add_child(xml_obj, answer):
             # Escape answer text before adding to xml tree.
             answer_text = six.text_type(Text(answer['text']))
-            child_str = u'<{tag_name} id="{id}">{text}</{tag_name}>'.format(
+            child_str = HTML('<{tag_name} id="{id}">{text}</{tag_name}>').format(
```

Comment:
Other option is to remove the previous line, and change this with something like
```python
            child_str = Text('{tag_begin}{text}{tag_end}'.format(
                 tag_begin=HTML('<{tag_name} id="{id}">'.format(
                     tag_name=self._child_tag_name, id=answer['id']
                 ),
                 text = str(answer['text']),
                 tag_end=HTML('</{tag_name}>'.format(tag_name=self._child_tag_name))
            )
```